MV.lua

```diff
@@ -0,0 +1,78 @@
+--[[ Module to perform matrix vector multiplication on two minibatch inputs,
+producing a minibatch.
+]]
+
+local MV, parent = torch.class('nn.MV', 'nn.Module')
+
+function MV:__init(trans)
+  parent.__init(self)
+
+  self.trans = trans or false
+
+  self.gradInput = {torch.Tensor(), torch.Tensor()}
+end
+
```

Comment:
local unpack  = unpack or table.unpack, and use that function in the rest of the module. this is for Lua back-compatibility reasons.